.github/workflows/lsp-ci.yaml

```diff
@@ -48,5 +48,5 @@ jobs:
             - name: Attach binaries
               uses: actions/upload-artifact@v3
               with:
-                name: langauge-server
-                path: lsp/awsdocuments-language-server-*
+                  name: langauge-server
+                  path: lsp/bin/awsdocuments-language-server-*
```

Comment:
I downloaded the artifacts and I see:

```
awsdocuments-language-server-linux
awsdocuments-language-server-macos
awsdocuments-language-server-win.exe
```

That is very verbose. The files could be named `aws-lsp-xx` instead without any ambiguity.

Reply:
This is the default behavior of the packager system. We will likely revisit the file names when we get closer to making final distribution bundles.

Reply:
For example, and TBD: we may have to differentiate x64 vs ARM binaries.